### PR TITLE
Add SQL Power Architect Community.app Cask

### PR DESCRIPTION
**Pull request by hanxue**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/sql-power-architect-community.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sql-power-architect-community' do
```
Comment (vitorgalvao) [inline Casks/sql-power-architect-community.rb]: You did not follow the token reference. This needs to be `sql-power-architect-jdbc`.

File: Casks/sql-power-architect-community.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version '1.0.8'
+  sha256 'a564e294aba5e7f4701717f8d36a898b06e6354000df332e5b6aa41377b06665'
+
+  url "http://download.sqlpower.ca/architect/1.0.8/community/SQL-Power-Architect-OSX-#{version}.tar.gz"
```
Comment (vitorgalvao) [inline Casks/sql-power-architect-community.rb]: You forgot to interpolate the first `1.0.8`.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 42d6dc02262885ddc86b332520d703ad43e93048. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by hanxue** [comment]
@vitorgalvao Thank you for pointing out the issues with the PR. Regarding the Cask token reference, I don't think it should follow the Mac application name of `SQL Power Architect JDBC`. The official product name is called `SQL Power Architect` by the vendor. [Product page](http://www.sqlpower.ca/page/architect)

There are 2 versions of SQL Power Architect [available for download](http://www.sqlpower.ca/page/architect_download_os):

1. Community edition
2. Enterprise Edition (trial)

Both versions will be installed as "SQL Power Architect JDBC". Similar to `intellij-idea` and `intellij-idea-ce`, this cask should be called `sql-power-architect-community`

@adidalal Thanks for fixing my mistake and accepting the PR :smile:

**Pull request closed without merging**